### PR TITLE
Implement spellcheck command

### PR DESCRIPTION
**Pull request by arkid15r**
### Describe Your Changes

Implement spellcheck command:
  - add cspell configuration files
  - dockerize spellchecking process
  - add Makefile targets
 
This PR adds a standalone  `make spellcheck-install` and `make spellcheck` targets to check `docs/*.{example,md}` files for spelling errors. The target process is dockerized to be run in a separate npm environment.

Some `docs/` typo fixes also included.

### Checklist

The following checks are **mandatory**:

- [x] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

File: cspell.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "docs/googlec3812dcf278679ec.html",
+    "go.mod",
+    "go.sum",
+    "lib/**",
+    "Makefile",
+    "package/**",
+    "ports/**",
+    "snap/**",
+    "vendor/**"
+  ],
```
Comment (hagen1778) [inline cspell.json:65]: I think we should simply limit spell check to `docs` folder. Then we can remove all these exclusions.

File: cspell/Makefile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+cspell-run-command:
+	@docker run \
+		--entrypoint /bin/sh \
+		--mount type=bind,src=".",dst=/victoriametrics \
+		--rm \
+		--tty \
+		cspell -c "$(CMD)"
+
+cspell-update: cspell-build
```
Comment (hagen1778) [inline cspell/Makefile:18]: Would you mind clarifying when this and `spellcheck-install` should be executed?

Comment (arkid15r) [reply cspell/Makefile:22]: I've added comments for each target. Please check if that's what you expected.

File: cspell.json (hunk 1 above)
Comment (arkid15r) [reply cspell.json]: That's a good point. I've fixed a misconfiguration in my code that allows to use this approach.
Thanks!

File: cspell.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (hagen1778) [inline cspell.json:1]: can we move this file to `cspell` folder?

File: cspell.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    "people-names",
+    "python",
+    "ru_RU",
+    "rust",
+    "software-terms",
+    "uk_UA",
+    "win32"
+  ],
+  "enabled": true,
+  "files": ["docs/**/*.{example,md}"],
```
Comment (hagen1778) [inline cspell.json:42]:
```suggestion
  "files": ["docs/**/*.md"],
```

I see no need in checking examples, as they contain go code. Do I miss something?

File: cspell.json (hunk 3 above)
Comment (arkid15r) [reply cspell.json]: Done.

File: cspell.json (hunk 4 above)
Comment (arkid15r) [reply cspell.json]: Done.

This is a leftover from [my first spellchecking PR](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6362/files#diff-82d4d91bd662440ab351943c532fc28affb11b181476485c34ea9e7325cf3dac) -- these files are under `docs` and also had some typos.

**Review by hagen1778 — APPROVED** [review]
Thank you!

**Comment by valyala** [comment]
@arkid15r , thanks for the great pull request!

**Comment by arkid15r** [comment]
> @arkid15r , could you add the same `make spellcheck` command to [VictoriaMetrics operator](https://github.com/VictoriaMetrics/operator/) repository, so it could fix errors at [operator docs](https://docs.victoriametrics.com/operator/)? cc @Amper and @Haleygo .

@valyala absolutely! It was in my plans to expand it to other repos once the PoC is accepted.

**Pull request merged**